src/ansys/aedt/core/filtersolutions_core/export_to_aedt.py

```diff
@@ -511,5 +838,861 @@ def export_design(self, export_format: ExportFormat.DIRECT_TO_AEDT, export_creat
+
+    @property
+    def substrate_cover_height_enabled(self) -> bool:
+        """Flag indicating if the substrate cover height is enabled.
+
+        Returns
+        -------
+        bool
+        """
+        substrate_cover_height_enabled = c_bool()
+        status = self._dll.getGroundedCoverAboveLine(byref(substrate_cover_height_enabled))
+        ansys.aedt.core.filtersolutions_core._dll_interface().raise_error(status)
+        return bool(substrate_cover_height_enabled.value)
+
+    @substrate_cover_height_enabled.setter
+    def substrate_cover_height_enabled(self, substrate_cover_height_enabled: bool):
+        status = self._dll.setGroundedCoverAboveLine(substrate_cover_height_enabled)
+        ansys.aedt.core.filtersolutions_core._dll_interface().raise_error(status)
+
+    def load_modelithics_models(self):
+        """Load ``Modelithics`` modesl from ``AEDT``."""
+        status = self._dll.loadModelitichsModels()
+        ansys.aedt.core.filtersolutions_core._dll_interface().raise_error(status)
+
+    @property
+    def modelithics_include_interconnect_enabled(self) -> bool:
+        """Flag indicating if the ``Modelithics`` interconnect inclusion is enabled.
+
+        Returns
+        -------
+        bool
+        """
+        modelithics_include_interconnect_enabled = c_bool()
+        status = self._dll.getModelithicsIncludeInterconnect(byref(modelithics_include_interconnect_enabled))
+        ansys.aedt.core.filtersolutions_core._dll_interface().raise_error(status)
+        return bool(modelithics_include_interconnect_enabled.value)
+
+    @modelithics_include_interconnect_enabled.setter
+    def modelithics_include_interconnect_enabled(self, modelithics_include_interconnect_enabled: bool):
+        status = self._dll.setModelithicsIncludeInterconnect(modelithics_include_interconnect_enabled)
+        ansys.aedt.core.filtersolutions_core._dll_interface().raise_error(status)
+
+    @property
+    def modelithics_inductor_list_count(self) -> int:
+        """Number of loaded ``Modelithics`` inductors.
+
+        Returns
+        -------
+        int
+        """
+        count = c_int()
+        status = self._dll.getModelithicsInductorsListCount(byref(count))
+        ansys.aedt.core.filtersolutions_core._dll_interface().raise_error(status)
+        return int(count.value)
+
+    def modelithics_inductor_list(self, row_index) -> str:
+        """Get the ``Modelithics`` inductor from the loaded list at the given index."""
+
+        modelithics_inductor_buffer = create_string_buffer(100)
+        status = self._dll.getModelithicsInductorsList(row_index, modelithics_inductor_buffer, 100)
+        ansys.aedt.core.filtersolutions_core._dll_interface().raise_error(status)
+        modelithics_inductor = modelithics_inductor_buffer.value.decode("utf-8")
+        return modelithics_inductor
+
+    @property
+    def modelithics_inductor_selection(self) -> str:
+        """Selected ``Modelithics`` inductor from the loaded list.
```

Comment:
Is this an inductor name or an inductor family name? The doc strings for inductor and the inductor family list need to clarify what both are an how to use them. My understanding is that the list in the GUI shows inductor family names and one family can be selected in the drop-down menu if a single family is suitable or multiple families can be added to the family list if the export should select from among multiple families. 

Whatever we do here to clarify for inductors should apply to the other part types as well.